my-dsl.md

```diff
@@ -2,37 +2,50 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+The language's name is ABC notation. [Link](http://abcnotation.com/)
 
 # Domain
 _Describe the language's domain in five words._
 
+Musical notation, especially folk, traditional
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a ContextFree program runs._
 
+The language itself is a plain-text ASCII representation of musical notes and symbols.  Various programs exist that parse these text files and generate TeX commands to ultimately create a musical score. Other output types are also possible.  [ABCJ](http://abcj.ganderband.com/), for example, is an ABC notation editor written in Java that parses ABC notation and outputs MIDI. 
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+With its narrow domain and limited functionality, ABC notation is purely domain specific. 
+
+ABC notation's intended use is to notate simple, single-voice tunes, though it is possible to (awkwardly) notate more complex pieces. In terms of functionality, an ABC notation "program" is simply a text file of ASCII characters, each character representing a musical symbol (eg. a note, tempo marking, accidental, measure line, etc). There are no loops or variables, and the language is not Turing complete, meaning that it cannot be classified as a general-purpose language.
+
+An issue Fowler may have with the classification of ABC notation as a DSL, however, is that it is not purely a "computer language," which according to him is a criterion of being a DSL (top of page 32). The [Wikipedia article](https://en.wikipedia.org/wiki/ABC_notation) on ABC notation (first paragraph) suggests that an ABC notated score can be an end product in itself, fully readable and shareable, even when written on a piece of paper or a napkin.
```

Comment:
Well if the napkin just contains a concept (the music score), can't you say the same about writing a piece of code on a napkin and giving it to someone? 

I do think it is a bit iffy that the language doesn't really compute much from the input given. It seems to just take the exact info given and just show it in graphical form. The user doesn't use language features to create something that didn't exist previously, since you are literally defining the exact things to show on the score. This is tricky... but I think its important to consider if computation is being used to expand the information and give the user more than they put in.